src/client/voice/VoiceConnection.js

```diff
@@ -117,6 +117,15 @@ class VoiceConnection extends EventEmitter {
   }
 
   /**
+   * The current dispatcher
+   * @type {?StreamDispatcher}
+   * @readonly
+   */
+  get dispatcher() {
+    return this.player.streams.size > 0 ? this.player.streams.last().dispatcher || null : null;
```

Comment:
Why specifically the last one? Why not the first? Or the 2nd? Or 3rd?

I don't really know how this is helpful at all, except adding a useless getter.